src/test/java/com/znsio/teswiz/businessLayer/vodqa/VodqaBL.java

```diff
@@ -152,4 +153,12 @@ public VodqaBL verifyLongPressedPopup() {
         assertThat(VodqaScreen.get().isLongPressedPopupVisible()).as("Long Pressed Popup is not visible").isTrue();
         return this;
     }
+
+    public VodqaBL pinchAndZoomOnAndElement() {
+        VodqaScreen vodqaScreen = VodqaScreen.get();
+        Dimension elementDimension = vodqaScreen.navigateToPhotoView().getImageElementDimension();
+        softly.assertThat(vodqaScreen.pinchAndZoomInOnAnElement().isPinchAndZoomInSuccessful(elementDimension)).as("Pinch and Zoom In on an element failed").isTrue();
+        softly.assertThat(vodqaScreen.pinchAndZoomOutOnAnElement().isPinchAndZoomInSuccessful(elementDimension)).as("Pinch and Zoom Out on an element failed").isTrue();
```

Comment:
Is there is a specific reason to use soft assert here?

Reply:
wanted to executed ahead even if any one of the asserts fail, as now these are separated into multiple methods, removed softly